packages/debugger/src/panels/sources/body.ts

```diff
@@ -43,13 +38,12 @@ export class SourcesBody extends Widget {
     });
     this._editor.hide();
 
-    this._model.currentFrameChanged.connect(async (_, frame) => {
-      if (!frame) {
+    this._model.currentSourceChanged.connect(async (_, currentSource) => {
+      if (!currentSource) {
         this._clearEditor();
         return;
       }
-
-      void this._showSource(frame);
+      void this._showSource();
     });
```

Comment:
It could be simplified I think, the source is already checked in `_showSource` now.
```ts
this._model.currentSourceChanged.connect(this._showSource);
```